pkg/config/cmd/cmd.go

```diff
@@ -222,14 +221,37 @@ func NewPrintNameOrErrorAfterIndent(mapper meta.RESTMapper, short bool, operatio
 	return func(info *resource.Info, err error) bool {
 		if err == nil {
 			fmt.Fprintf(out, indent)
-			cmdutil.PrintSuccess(mapper, short, out, info.Mapping.Resource, info.Name, dryRun, operation)
+			printSuccess(mapper, short, out, info.Mapping.Resource, info.Name, dryRun, operation)
 		} else {
 			fmt.Fprintf(errs, "%s%s: %v\n", indent, prefixForError(err), err)
 		}
 		return false
 	}
 }
 
+func printSuccess(mapper meta.RESTMapper, shortOutput bool, out io.Writer, resource, name string, dryRun bool, operation string) {
```

Comment:
Ha, you're adding this method here but also in `UPSTREAM: <carry>: add back PrintSucces`. Drop the carry and make this public somewhere in `pkg/cmd/util`.

Reply:
> Ha, you're adding this method here but also in UPSTREAM: <carry>: add back PrintSucces. Drop the carry and make this public somewhere in pkg/cmd/util.

Eh, stuff got thrashy as @juanvallejo  tried to make test-cmd pass.  Given what we're about to do and how bad the printing stack is, I honestly don't care what happens, but I don't want to twiddle with it much here.